geomesa-tools/conf/log4j.properties

```diff
@@ -4,7 +4,7 @@ log4j.rootLogger=info,file
 geomesa.log.dir=.
 
 # set logging levels to appropriate values
-log4j.logger.org.locationtech.geomesa.tools=info,stderr
+log4j.logger.org.locationtech.geomesa.tools=info
```

Comment:
ah, i think we should keep stderr actually - i think we use it for messages to the user